### PR TITLE
Add release workflow and document

### PR DESCRIPTION
**Pull request by morimoto-cybozu**
Signed-off-by: morimoto-cybozu <kenji_morimoto@cybozu.co.jp>

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    name: Release on GitHub
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: Create release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          version=$(echo ${{ github.ref }} | sed -e "s#refs/tags/##g")
+          if echo $version | grep -q -e '-'; then prerelease=-p; fi
```
Comment (zeroalphat) [inline .github/workflows/release.yaml:17]: Do I understand correctly that if the TAG contains a `-`, it becomes prelease?

Comment (morimoto-cybozu) [reply .github/workflows/release.yaml:17]: @zeroalphat 
Yes.
A normal tag should be formed like "x.y.z" and does not contain hyphens.
A use case of a hyphen is `1.xx.0-rc.1` in the following document.
https://neco.cybozu.wiki/procedure_k8s_upgrade/#cke

cf. https://semver.org/#spec-item-9

Comment (zeroalphat) [reply .github/workflows/release.yaml:17]: Thank you for your response.
It helped me to understand.

**Review by zeroalphat — APPROVED** [review]
LGTM

**Pull request merged**